src/Engine/Spells/CastSpellInfo.cpp

```diff
@@ -3227,3 +3187,12 @@ void pushNPCSpell(SPELL_TYPE spell) {
 void pushScrollSpell(SPELL_TYPE spell, unsigned int uPlayerID) {
     pushSpellOrRangedAttack(spell, uPlayerID, SCROLL_OR_NPC_SPELL_SKILL_VALUE, ON_CAST_CastViaScroll, 0);
 }
+
+void spellTargetPicked(int pid, int playerTarget) {
+    CastSpellInfo *pCastSpell = static_cast<CastSpellInfo *>(pGUIWindow_CastTargetedSpell->wData.ptr);
```

Comment:
Either pid or playerTarget should be set when calling this function, right? Let's add an assert to check that.